lib/rubocop/cop/layout/space_around_operators.rb

```diff
@@ -170,6 +170,19 @@ def offense(type, operator, with_space, right_operand)
           yield msg if msg
         end
 
+        def enclose_operator_in_space(corrector, range)
```

Comment:
I think it's `enclose_with`, not `enclose_in`.

Reply:
I updated this method name. Thank you!